spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/util/ObservableReactiveUtil.java

```diff
@@ -68,6 +72,21 @@ private ObservableReactiveUtil() {
 		});
 	}
 
+	public static <ResponseT, RequestT, A> Flux<ResponseT> streamingBidirectionalCall(
```

Comment:
Maybe use `T` instead of `A` to represent the actual data type; it's more conventional.